source/admin_manual/submission_server.rst

```diff
@@ -5,22 +5,20 @@ Submission Server
 =================
 
 As of version 2.3.0, Dovecot provides an SMTP submission service, also known as
-a `Mail Submission Agent (MSA) <https://tools.ietf.org/html/rfc6409>`_. It is
+a Mail Submission Agent (MSA) :rfc:`6409`. It is
 currently implemented as a proxy that acts as a front-end for any :ref:`mta`,
 adding the necessary functionality required for a submission service: it adds
-the required `AUTH <https://tools.ietf.org/html/rfc4954>`_ support, avoiding
+the required AUTH :rfc:`4954` support, avoiding
 the need to configure the MTA for :ref:`SASL authentication <sasl>`. More SMTP
-capabilities like `CHUNKING <https://tools.ietf.org/html/rfc3030>`_ and `SIZE
-<https://tools.ietf.org/html/rfc1870>`_ are supported, without requiring the
-backend MTA supporting these extensions. Other capabilities like `8BITMIME
-<https://tools.ietf.org/html/rfc6152>`_ and `DSN
-<https://tools.ietf.org/html/rfc3461>`_ currently require support from the
+capabilities like CHUNKING :rfc:`3030` and SIZE :rfc:`1870` are supported, without requiring the
+backend MTA supporting these extensions. Other capabilities like 8BITMIME :rfc:`6152` and
+DS :rfc:`3461` currently require support from the
 backend/relay MTA.
```

Comment:
Re-flow text. The paragraph looks visually unraveled now with the removed links.